utest/output/test_pylogging.py

```diff
@@ -0,0 +1,141 @@
+import unittest
+
+from robot.utils.asserts import assert_equal, assert_true, assert_false
+
+import robot.output.pyloggingconf as pyLogging
+
+import logging
+
+
+class MessageMock:
+    def __init__(self, timestamp, level, message):
+        self.timestamp = timestamp
+        self.level = level
+        self.message = message
+
+
+class MockLibraryLogger:
+    def __init__(self):
+        self.last_message = (None, None)
+
+    def error(self, message):
+        self.last_message = (message, logging.ERROR)
+
+    def warn(self, message):
+        self.last_message = (message, logging.WARNING)
+
+    def info(self, message):
+        self.last_message = (message, logging.INFO)
+
+    def debug(self, message):
+        self.last_message = (message, logging.DEBUG)
+
+    def trace(self, message):
+        self.last_message = (message, logging.NOTSET)
+
+
+class TestPyLogging(unittest.TestCase):
+    def setUp(self):
+        self.library_logger = MockLibraryLogger()
+        self.test_handler = pyLogging.RobotHandler(library_logger=self.library_logger)
+        root = logging.getLogger()
+        root.setLevel(logging.NOTSET)
+        for handler in root.handlers:
+            root.removeHandler(handler)
+        root.addHandler(self.test_handler)
+
+    def tearDown(self):
+        root = logging.getLogger()
+        root.removeHandler(self.test_handler)
+
+    def test_python_logging_debug(self):
+        TESTMESSAGE = "This is a test message"
```

Comment:
Using `ALLCAPS` with local variables is a bit odd and against PEP-8. Could you change them? This particular message is used multiple times so perhaps there could be `test_message = 'This is a test message'` on the class level so that tests could just use `self.test_message`.